### PR TITLE
Ensure backward compatibility

### PR DESCRIPTION
**Pull request by dawid-pstrak-rmtcfm-com**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -14,34 +14,34 @@
...
+        "cors": "2.8.5",
+        "dotenv": "7.0.0",
+        "express": "4.16.4",
+        "express-async-errors": "3.1.1",
+        "express-session": "1.15.6",
+        "express-validator": "6.12.0",
+        "helmet": "3.16.0",
+        "http-status-codes": "1.3.2",
+        "lodash": "4.17.11",
+        "moment": "2.22.2",
```
Comment (rafal-ksiazek-rmtcfm-com) [inline backend/package.json]: dayjs

File: backend/package.json [hunk 2]
```diff
@@ -14,34 +14,34 @@
     "jest": {},
     "license": "MIT",
     "dependencies": {
-        "bcryptjs": "^2.4.3",
+        "bcryptjs": "2.4.3",
```
Comment (rafal-ksiazek-rmtcfm-com) [inline backend/package.json]: ~ instead of ^

File: backend/package.json (hunk 1 above)
Comment (dawid-pstrak-rmtcfm-com) [reply backend/package.json]: moment was unused dep. done

File: backend/package.json (hunk 2 above)
Comment (dawid-pstrak-rmtcfm-com) [reply backend/package.json]: done

**Pull request merged**